app/src/main/res/layout/activity_movie_details.xml

```diff
@@ -0,0 +1,284 @@
+                android:text="@string/feedback_rate"
+                android:textColor="@color/tint_color"
+                android:textSize="14dp"
+                android:fontFamily="@font/robotoblack"
+                android:layout_marginTop="4dp"
+                app:layout_constraintStart_toEndOf="@id/fifth_Star"
+                app:layout_constraintTop_toBottomOf="@id/film_genre"/>
+            <TextView
+                android:id="@+id/storyline"
+                android:layout_width="0dp"
+                android:layout_height="wrap_content"
+                android:layout_marginTop="48dp"
+                android:layout_marginBottom="4dp"
+                android:text="@string/storyline"
+                android:textColor="@color/text_color"
+                android:textStyle="bold"
+                android:textSize="14dp"
+                android:fontFamily="@font/robotoblack"
+                app:layout_constraintStart_toStartOf="@id/first_Star"
+                app:layout_constraintTop_toBottomOf="@id/film_genre"
+                app:layout_constraintEnd_toEndOf="@id/description"/>
+            <TextView
+                android:id="@+id/description"
+                android:layout_width="0dp"
+                android:layout_height="wrap_content"
+                android:text="@string/film_description"
+                android:textColor="@color/text_color_opacity"
+                android:textSize="14dp"
+                android:fontFamily="@font/robotolight"
+                android:layout_marginTop="4dp"
+                android:paddingTop="6dp"
+                android:paddingBottom="6dp"
+                app:layout_constraintStart_toStartOf="@id/storyline"
+                app:layout_constraintTop_toBottomOf="@id/storyline"
+                app:layout_constraintEnd_toEndOf="parent"
+                android:layout_marginEnd="16dp"/>
+            <TextView
+                android:id="@+id/cast_Text_View"
+                android:layout_width="0dp"
+                android:layout_height="wrap_content"
+                app:layout_constraintHorizontal_weight="1"
+                android:text="@string/cast"
+                android:textColor="@color/white"
+                android:textStyle="bold"
+                android:textSize="14dp"
+                android:fontFamily="@font/robotoblack"
+                android:paddingBottom="7dp"
+                android:layout_marginTop="24dp"
+                app:layout_constraintEnd_toEndOf="@id/description"
+                app:layout_constraintStart_toStartOf="@id/description"
+                app:layout_constraintTop_toBottomOf="@id/description"/>
+
+            <ImageView
+                android:id="@+id/first_actor_image_view"
+                android:layout_width="0dp"
+                android:layout_height="0dp"
+                android:src="@drawable/img_one"
+                app:layout_constraintDimensionRatio="1:1"
+                app:layout_constraintHorizontal_weight="1"
```

Comment:
remove
app:layout_constraintHorizontal_weight="1"
app:layout_constraintHorizontal_chainStyle="spread_inside"
for all actors images